api/managementpb/checks.proto

```diff
@@ -7,6 +7,14 @@ option go_package = "api/managementpb;managementpb";
 import "google/api/annotations.proto";
 import "managementpb/severity.proto";
 
+// SecurityCheckInterval represents possible execution interval values for checks.
+enum SecurityCheckInterval {
+  SECURITY_CHECK_INTERVAL_INVALID = 0;
+  STANDARD = 1;
+  FREQUENT = 2;
```

Comment:
Can we add a short description for each interval, e.g. from where they come?

Reply:
will add it

Reply:
on second thoughts all the intervals come from the same source (the UI in this case) so why do we need a comment for each of them. could you give me example of what you are looking for?